dev/DynamicDependency/Powershell/Delete-PackageDependency.ps1

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation and Contributors.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+TODO
+#>
+[CmdletBinding(SupportsShouldProcess=$true)]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$PackageDependencyId
+)
+
+Set-StrictMode -Version 3.0
+
+$ErrorActionPreference = "Stop"
+
+# Import the MSIX Dynamic Dependency module
+if (-not (Get-Module | Where-Object {$_.Name -eq 'MsixDynamicDependency'}))
+{
+    $module = Join-Path $PSScriptRoot 'MsixDynamicDependency.psm1'
+    Import-Module -Name $module -ErrorAction Stop
```

Comment:
Unclear to me why not "-ErrorAction $ErrorActionPreference"?

Reply:
These are intentional, though perhaps they shouldn't be?

Good for now but I'll talk to some folks and if it should be changed I'll make a new PR incorporating that